pybossa/view/projects.py

```diff
@@ -1498,3 +1495,20 @@ def auditlog(short_name):
                            n_task_runs=n_task_runs,
                            n_completed_tasks=cached_projects.n_completed_tasks(project.get('id')),
                            n_volunteers=cached_projects.n_volunteers(project.get('id')))
+
+
+@blueprint.route('/<short_name>/publish', methods=['GET', 'POST'])
+@login_required
+def publish(short_name):
+    (project, owner, n_tasks, n_task_runs,
+     overall_progress, last_activity) = project_by_shortname(short_name)
+
+    ensure_authorized_to('publish', project)
```

Comment:
yes, that handles the permissions over the action publish for a given project and user